playbooks/de-rm-containers-cleanup.yaml

```diff
@@ -45,3 +45,41 @@
       ignore_errors: yes
       become: true
       shell: docker rm -v config_jexevents
+
+- name: Remove anon-files and anon-files-nginx (CORE-8534)
+  hosts: systems
+  become: true
+  gather_facts: false
+  tags:
+    - services
+    - kifshare
+    - anon-files
+  post_tasks:
```

Comment:
This part could probably be done in some fancy way with docker tags.

But it didn't seem worth the time when it's a small list of known names stuff will appear under.